### PR TITLE
Implemented method to choose threshold and improved free command

### PR DESCRIPTION
**Pull request by Kenlhc**
Feel free to comment.

### CODE REVIEW

File: src/main/java/oof/Storage.java [hunk 1]
```diff
@@ -49,6 +50,39 @@
...
+    public int readThreshold() throws OofException {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(PATH_THRESHOLD));
+            String line;
+            line = reader.readLine();
+            int threshold;
+            threshold = Integer.parseInt(line);
+            return threshold;
+        } catch (IOException e) {
+            throw new OofException("File Unavailable!");
```
Comment (kahlokkee) [inline src/main/java/oof/Storage.java]: Unnecessary to throw an exception here, just create a default config file

File: src/main/java/oof/Reminder.java [hunk 2]
```diff
@@ -14,8 +16,8 @@
  */
 public class Reminder {
 
+    private Storage storage = new Storage();
```
Comment (kahlokkee) [inline src/main/java/oof/Reminder.java]: Can use storage object instantiated in Oof constructor

**Review by kahlokkee — CHANGES_REQUESTED** [review]

File: src/main/java/oof/Storage.java (hunk 1 above)
Comment (jasperosy) [reply src/main/java/oof/Storage.java]: Issue resolved.

**Comment by jasperosy** [comment]
Looks good. Waiting for @KahLokKee to approve the changes and merge the PR.

**Review by kahlokkee — APPROVED** [review]

**Pull request merged**